src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let finalPrice = 0;
+
+  if (days >= 7) {
+    finalPrice = 40 * days - 50;
```

Comment:
You can return the calculated price directly from within the `if` block to make the code more concise and to avoid unnecessary `else` blocks.